_data/speakers.yml

```diff
@@ -7,3 +7,6 @@
 - name: Naomi Ceder
   brief: Trans*Code<br/>Python Software Foundation
   image: /assets/images/naomiceder_headshot_web.jpg
+- name: James Powell
+  brief: NumFOCUS<br/>PyData
+  image: /assets/images/jamespowell.jpg
```

Comment:
@sayanchowdhury We discussed this. Please sort this.

From the comment above,
> # NOTE: Please ensure the names are lexically sorted.

Reply:
Sorry, did this, but did not add the changes.